experiments/ClimaEarth/components/ocean/oceananigans.jl

```diff
@@ -343,7 +346,7 @@ function FieldExchanger.update_sim!(sim::OceananigansSimulation, csf, area_fract
     oc_flux_S = surface_flux(sim.sim.model.tracers.S)
     precipitating_fresh_water_flux = (remapped_P_liq .+ remapped_P_snow) ./ ocean_fresh_water_density
     surface_salinity_flux = OC.interior(sim.sim.model.tracers.S, :, :, 1) .* precipitating_fresh_water_flux
-    view(oc_flux_S, :, :, 1) .= .-surface_salinity_flux
+    oc_flux_S .= .-surface_salinity_flux
```

Comment:
```suggestion
    @. oc_flux_S = -surface_salinity_flux
```

Reply:
`oc_flux_S` is an Oceananigans Field. What would you recommend doing instead of broadcasting?

Reply:
Then unwrap:

```julia
interior(oc_flux_S, :, :, 1) .= .-surface_salinity_flux
```

Reply:
Ahh okay, is this why we were using `view` before?